hpc_plugin/tasks.py

```diff
@@ -318,7 +338,16 @@ def deploy_job(script,
             "' not supported.")
 
     # Execute the script and manage the output
+    ctx.logger.info('Creating shell script..')
     client = SshClient(credentials)
+    ctx.logger.info('deploy_job: script=%s'%script)
+    if 'revert' in script:
+        # ctx.logger.info('here we are in revert')
+        exit_code = wm.send_notify_user(client,
```

Comment:
Notification to a new user should be done in an independent task, and run by the run_jobs workflow when it ends. Here is really notifying the user when a revert is performed.